README.md

```diff
@@ -1,470 +1,74 @@
-  ref_scenario,
-  method="collaborative",
-  targets_filename="eastern_2030_clean_energy_targets.csv",
-  addl_curtailment=addl_curtailment,
-  next_scenario=new_scenario,
-  solar_fraction=0.55
-)
-```
-
-
-### D. Creating a Change Table from Capacity Planning Results
-The capacity planning framework returns a data frame of capacities by resource type and target area, but the Scenario creation process ultimately requires scaling factors by resource type and zone or plant_id. A function `create_change_table` exists to perform this conversion process. Using a reference scenario, a set of scaling factors by resource type, zone, and plant_id is calculated. When applied to a base `Grid` object, these scaling factors will result in capacities that are nearly identical to the reference scenario on a per-plant basis (subject to rounding), with the exception of solar and wind generators, which will be scaled up to meet clean energy goals.
-```python
-from powersimdata.design.generation.clean_capacity_scaling import create_change_table
-
-change_table = create_change_table(targets_and_new_capacities_df, ref_scenario)
-# The change table method only accepts zone names, not zone IDs, so we have to translate
-id2zone = new_scenario.state.get_grid().id2zone
-# Plants can only be scaled one resource at a time, so we need to loop through
-for resource in change_table:
-	new_scenario.state.builder.change_table.scale_plant_capacity(
-		resource=resource,
-		zone_name={
-			id2zone[id]: value
-			for id, value in change_table[resource]["zone_name"].items()
-		},
-		plant_id=change_table[resource]["zone_name"]
-	)
-```
-
-
-## 5. Analyzing Scenario Designs
-### A. Analysis of Transmission Upgrades
-#### I. Cumulative Upgrade Quantity
-Using the change table of a scenario, the number of upgrades lines/transformers and their cumulative upgraded capacity (for transformers) and cumulative upgraded megawatt-miles (for lines) can be calculated with:
-```python
-powersimdata.design.transmission.mwmiles.calculate_mw_miles(scenario)
-```
-where `scenario` is a `Scenario` instance.
-
-
-#### II. Classify Upgrades
-The upgraded branches can also be classified into either interstate or intrastate branches by calling:
-```python
-powersimdata.design.transmission.statelines.classify_interstate_intrastate(scenario)
-```
-where `scenario` is a `Scenario` instance.
-
-
-### B. Analysis of Generation Upgrades
-#### I. Accessing and Saving Relevant Supply Information
-Analyzing generator supply and cost curves requires the proper generator cost and plant information to be accessed from a Grid object. This data can be accessed using the following:
-```python
-from powersimdata.design.generation.cost_curves import get_supply_data
-
-supply_df = get_supply_data(grid, num_segments, save)
+Only a limited set of feature is available when solely installing **PowerSimData**. If you chose this option, we recommend that you use `pipenv`:
```

Comment:
"..set of features are available..", and "If you choose this option"

Reply:
Done